README.md

```diff
@@ -20,7 +20,8 @@ Update Foreman with the new gems:
 # Configuration
 
 By default the plugin will display the Domain associated by each host. This is not
-massively useful. To set your own choice of column, add this to Foreman's config file
+massively useful. To set your own choice of column, add this to Foreman's plugin config file,
+for example /etc/foreman/plugins/foreman_column_view.yaml
```

Comment:
Yeah, this is because the plugin was written before plugins had their own config dir - back then you actually had to add config to the main Foreman config file :)

This will be correct for package installs, but not source installs - could you add "or /path/to/your/foreman/config/settings.plugins.d/foreman_column_view.yaml" or some similar wording?